src/hooks/use-mobile-view/useMobileView.ts

```diff
@@ -0,0 +1,8 @@
+import { em } from '@mantine/core';
+import { useMediaQuery } from '@mantine/hooks';
+
+const smMobileViewMaxWidth = 768;
```

Comment:
I guess we need to use our variables for that. Now we have breakpoints in our variables.scss. Let's try to add custom properties for them and after that we can reuse variables in code. In addition, I propose to add this variables to mantine theme, so we will get single source of truth
```
const theme = createTheme({
  breakpoints: {
    xs: '30em',
    sm: '48em',
    md: '64em',
    lg: '74em',
    xl: '90em',
  },
});
<MantineProvider theme={theme}>
    {/* Your app here */}
</MantineProvider>
```

https://mantine.dev/styles/responsive/

Reply:
I investigated the issue a little bit. It looks like we can't use custom properties here. So, for now I propose to create the same variables for breakpoints in javascript that copy values from variables.scss and use them in UseMobileView. Also, I guess you can add these variables to mantine theme, just to init them explicitly. Later we will try to merge scss and js breakpoints (e.g. using post csss plugin)